jwala-common/src/main/java/com/cerner/jwala/common/rule/jvm/JvmNameRule.java

```diff
@@ -16,8 +21,13 @@ protected MessageResponseStatus getMessageResponseStatus() {
     }
 
     @Override
+    public boolean isValid() {
+        return super.isValid() && name.length() <= MAX_LENGTH;
+    }
+
+    @Override
     protected String getMessage() {
-        return "Invalid Jvm Name : \"" + name + "\"";
+        return name != null && name.length() > MAX_LENGTH ? "Length of JVM should not exceed " + MAX_LENGTH + " characters" : "Invalid Jvm Name : \"" + name + "\"";
```

Comment:
I would add instructions on how to increase the max length using the new property here as well - just in case the client is using a load balancer that supports more than 64 characters for the route name